docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -709,6 +688,12 @@ Assumptions and Limitations
 #. We assume that MAM treatment coverage is equal to SAM treatment coverage. Given that SAM treatment is more intensive than MAM treatment, we may underestimate MAM treatment coverage as a result of this assumption.
 #. We assume that MAM and SAM treatment effectivenesses are independent from one another.
 #. We assume that individual simulant's propensity to respond to wasting treatment is independent of their previous response/non-response to treatment. According to [Zw_2020tx]_, SAM treatment response rates are associated with diarrhea, oedema, and use of antibiotics in the treament course in Ethiopia. Additionally, vitamin A supplementation and distance from the treatment center may be associated with SAM treatment response rates, although direct evidence was not provided [Zw_2020tx]_. We chose to make this assumption given the non-deterministic nature of these factors.
+#. We assume that MAM and SAM treatment non-responders have the same time to recovery as untreated MAM and SAM cases.
```

Comment:
What does this mean exactly? It sounds self-contradictory at first glance, so I don't think I understand...

Reply:
Are you saying that the remission rate (SAM -> MAM or MAM -> MILD) of non-responders is the same as the remission rate of untreated cases? I think the wording "time to recovery" was throwing me off because it sounds like they are recovering (e.g. SAM -> MILD) even though they don't respond to treatment.

Reply:
So the studies that investigated time to recovery defined non-responders as treated children who had not recovered (no longer wasted) within two months of treatment. They censored non-responders along with those who died or were lost to follow up in the estimation of time to recovery. Since we don't have information on transition rates among this group, we are treating them as if they were not treated at all (untreated SAM to MAM recovery rate or untreated MAM to mild recovery rate, depending on which state they were in initially), which is likely not actually the case. I will expand upon this in the limitation.